packages/psss/src/components/Table/CellComponents/PercentageChangeCell.js

```diff
@@ -21,11 +21,17 @@ const SuccessStyleText = styled.span`
  * Displays a value as a percentage
  */
 export const PercentageChangeCell = ({ percentageChange, className }) => {
-  if (percentageChange > 0) {
-    return <WarningStyleText className={className}>+{percentageChange}%</WarningStyleText>;
+  if (!percentageChange) {
+    return '-';
```

Comment:
Just want to check that this means that if there's no percentage change in the data payload then we will display `-`? If so this is correct, since sometimes there is no percentage change (ie. if the previous week had 0 cases, we won't display a percentage change).

However, we may want to differentiate between a valid `0%` percentage change and a non-existant percentage change?

```suggestion
  if (percentageChange === undefined) {
    return '-';
```

Reply:
Yes - great catch! I totally agree